### PR TITLE
Update ruckus_unleashed.markdown

### PR DESCRIPTION
**Pull request by pathia**
Added the requirement to add the ip address of the master AP.
It was not mentioned in the documentation and took me some time to tackle. I just don't want other to make the same mistake.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ruckus_unleashed.markdown [hunk 1]
```diff
@@ -31,6 +31,7 @@ You will have to create a user on the device which is a **Monitoring Admin**. Lo
 ## Troubleshooting
 
 For this platform to work, the Ruckus Unleashed device will need to be accessible over SSH. If you are having trouble with Home Assistant not connecting, make sure the user you created above can log in to SSH and can run privileged commands.
+Remember to configure the IP address of your master access point. See Access Points on the management web interface. And perhaps consider to set a preferred master (Admin & Services>System>System Info>Preferred master)
```
Comment (frenck) [inline source/_integrations/ruckus_unleashed.markdown]: Why is this part of the after fact/troubleshooting and not part of the configuration instructions?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by pathia** [comment]
Added the requirement to add the IP address of the master AP. Now in the proposed section.

File: source/_integrations/ruckus_unleashed.markdown [hunk 2]
```diff
@@ -20,6 +20,7 @@ There is currently support for the following device types within Home Assistant:
 ## Configuration
 
 To add a Ruckus Unleashed device to your installation, go to **Configuration** -> **Integrations**, click the `+` button, then select **Ruckus** from the list of integrations.
+It is required to configure the IP address of your **master access point**. See the section Access Points on the management web interface. And perhaps consider to set a preferred master (Admin & Services>System>System Info>Preferred master)
```
Comment (frenck) [inline source/_integrations/ruckus_unleashed.markdown]:
```suggestion

It is required to configure the IP address of your **master access point**. See the section Access Points on the management web interface. And perhaps consider to set a preferred master (Admin & Services>System>System Info>Preferred master).
```

**Review by frenck — APPROVED** [review]
Thanks, @pathia 👍

**Pull request merged**